runtime/src/lib.rs

```diff
@@ -256,10 +259,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("sora-substrate"),
     impl_name: create_runtime_str!("sora-substrate"),
     authoring_version: 1,
-    spec_version: 71,
+    spec_version: 72,
     impl_version: 1,
     apis: RUNTIME_API_VERSIONS,
-    transaction_version: 71,
+    transaction_version: 72,
```

Comment:
Can we update the spec version on the `develop` branch first of all? And then merge it into the `master`